docs/getting-started/velaux-concept.md

```diff
@@ -2,4 +2,4 @@
 title: VelaUX Concept
 ---
 
-This doc has migrated to [velaux addon introdution](../reference/addons/velaux#concept-of-velaux).
\ No newline at end of file
+This doc has migrated to [velaux addon introdution](../reference/addons/velaux.md#concept-of-velaux).
```

Comment:
seems `.md` suffix is not necessary?

Reply:
Pls correct if I'm wrong: if we don't add the suffix, when viewing the markdown file, the attached link [docs/getting-started/velaux-concept.md](https://github.com/kubevela/kubevela.io/blob/main/docs/reference/addons/velaux#concept-of-velaux) is invalid.

I attach the error message here

![Screenshot 2022-11-29 at 20 56 15](https://user-images.githubusercontent.com/3270746/204547920-db3b4bb3-8701-45b8-aab2-75e783912ab6.png)

Reply:
Seems like @anhldbk is viewing it in GitHub instead of kubevela.io. Thanks @anhldbk but this repo is mainly focused on being rendered by docusaurus and viewed in kubevela.io

Reply:
@chivalryq Correct. I'm gonna close this pull request. Thanks for your supports.